src/sagas/nanoContract.js

```diff
@@ -107,9 +107,24 @@ export function* registerNanoContract({ payload }) {
   yield put(nanoContractRegisterSuccess(nc));
 }
 
+/**
+ * Updates nano contract registered address in the wallet storage
+ * @param {{
+ *   payload: {
+ *     address: string,
+ *     ncId: string,
+ *   }
+ * }} action with request payload.
```

Comment:
```suggestion
 * @param {Object} state
 * @param {Object} action with request payload.
 * @param {string} action.payload.ncId
 * @param {string} action.payload.address
```

Reply:
Improved the docstring https://github.com/HathorNetwork/hathor-wallet/pull/623/commits/d9e3f1cb964ed8bb849bf018fc3ab075e235a7ad